samples/wifi/shell/src/wifi_shell.c

```diff
@@ -0,0 +1,341 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/* @file
+ * @brief WPA supplicant shell module
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+
+#include <zephyr/zephyr.h>
+#include <zephyr/shell/shell.h>
+#include <zephyr/init.h>
+
+#include <net/net_if.h>
+#include <net/wifi_mgmt.h>
+#include <net/net_event.h>
+#include <zephyr_fmac_main.h>
+
+#ifdef CONFIG_WPA_SUPP
+#include <src/utils/common.h>
+#include <wpa_supplicant/config.h>
+#include <wpa_supplicant/wpa_supplicant_i.h>
+#endif /* CONFIG_WPA_SUPP */
+
+#define SUPPLICANT_SHELL_MODULE "wpa_cli"
```

Comment:
```suggestion
#define WIFI_SHELL_MODULE "wifi_cli"
```

Reply:
There is a external dependency to leave this as wpa_cli, as we are planning to use some Linux apps for Wi-Fi certification and they all rely on this name `wpa_cli`, and also our QA (and in general any wifi QA) are familiar with this and have shell scripts and automation based on this, so, using this is preferred.

Reply:
Could you add this rationale in a comment close to this macro?

Reply:
Removed this unused macro, but added a comment where we do `SHELL_CMD_REGISTER` with wpa_cli as the name.